shell/pages/c/_cluster/settings/performance.vue

```diff
@@ -189,11 +199,15 @@ export default {
       });
     },
 
-    setPaginationDefaults() {
+    sspApplyDefaults(defaultStore) {
```

Comment:
I understand what you're doing here but from a user perspective I find how we apply defaults jarring.


https://github.com/user-attachments/assets/24653196-610a-410f-9bfc-6aa55cd374dd

I expected to set the resources to the default not all the settings. I wanted to look at the resources to see if anything changed but everything got disabled and closed.

Reply:
This was just a convenience thing for development (gated on isDev), removed